Orange/widgets/model/owsgd.py

```diff
@@ -76,7 +76,7 @@ class Outputs(OWBaseLearner.Outputs):
     learning_rate_index = Setting(0)
     eta0 = Setting(.01)
     power_t = Setting(.25)
-    n_iter = Setting(5)
+    max_iter = Setting(10)
```

Comment:
Default in sklearn is 5. Is there a particular reason to have it at 10?

Reply:
No particular reason. Sklearn will change it to 1000 from 0.21. I decided for 10 only to see if I changed everything correctly (testing widget in Orange).